src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -1,188 +1,169 @@
 # rubocop:disable Metrics/ClassLength
```

Comment:
The changes in this file are mostly indentation changes. To make the review easier, you can hide whitespace changes. See this blog article for details on how to do this: https://cloudfour.com/thinks/quick-tip-how-to-hide-whitespace-changes-in-git-diffs/

Reply:
Good to know. Are you done with the updates in this PR?

Reply:
Yes :slightly_smiling_face:

Reply:
Thanks a lot :)